cmd/kwil-admin/cmds/node/version.go

```diff
@@ -12,7 +12,7 @@ var (
 	versionLong = `Print the node's version information. The version is the Kwil's version string, set at compile time.`
 
 	versionExample = `# Print the node's version information
-kwil-admin node version --rpcserver localhost:50151 --authrpc-cert "~/.kwild/rpc.cert"`
+kwil-admin node version --rpcserver /tmp/kwild.socket`
```

Comment:
nbd, seems you gave `--rpcserver` to different example values in other cmds.  Maybe use same example value?